fe/src/app/advert/page.tsx

```diff
@@ -0,0 +1,184 @@
+"use client"
+
+import { useState } from "react";
+import toast from "react-hot-toast";
+import Datepicker, { DateValueType } from "react-tailwindcss-datepicker"; 
+import YesOrNoChoice from "@/components/YesOrNoChoice/YesOrNoChoice";
+import { Button, Input, Tab, TabPanel, Tabs, TabsBody, TabsHeader, Textarea } from "@/lib/materialTailwindExports";
+import { QuestionsFormType, SimpleFormType } from "@/types/CreateAdvertForm";
+import { FORM_BOX_SHADOW } from "@/utils/constants";
+import { QUESTIONS_FORM_TYPE, SIMPLE_FORM_TYPE, questionsFormDefaultValues, questionsFormSchema, simpleFormDefaultValues, simpleFormSchema } from "./utils";
+import { USER_ADVERTS_ROUTE } from "@/utils/urls";
+import ImageDropzone from "@/components/ImageDropzone/ImageDropzone";
+
+const AdvertPage = () => {
+  const [formType, setFormType] = useState<string>(SIMPLE_FORM_TYPE);
+  const [questionsFormData, setQuestionsFormData] = useState<QuestionsFormType>(questionsFormDefaultValues);
+  const [simpleFormData, setSimpleFormData] = useState<SimpleFormType>(simpleFormDefaultValues);
+
+  const handleEndDateChange = (date: DateValueType) => {
```

Comment:
I think we also need advert finish time and hours here. It is not convenient for a client that his advert finishes 12pm at midnight.

Reply:
I would argue with that as in my opinion its okay that the advert will be open for offers until it has passed this date instead of taking the exact creation date. What are your opinions? @MatasAdmoneo @Saulenas69 @AthenaKas